tests/utils/verification/volumeproperties.go

```diff
@@ -150,3 +150,13 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+
+// GetPolicyNameForVol returns the vsan policy name used by the volume using docker cli
+func GetPolicyNameForVol(hostname string, volName string) string {
+	cmd := dockercli.InspectVolume + " --format '{{index .Status \"vsan-policy-name\"}}' " + volName
+	op, _ := ssh.InvokeCommand(hostname, cmd)
```

Comment:
is there a reason for ignoring error field? Util may return err field to testcase and let test case to fail with proper message.